src/Resolvers/Microsoft.DotNet.WorkloadManifestReader/WorkloadManifest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Microsoft.NET.Sdk.WorkloadResolver
+{
+    class WorkloadManifest
+    {
+        public Dictionary<string, List<string>> Workloads { get; set; } = new Dictionary<string, List<string>>(StringComparer.OrdinalIgnoreCase);
```

Comment:
The interface is the same as the real thing or a place holder too?

Reply:
I'm sure the API will change.